Gemfile.lock

```diff
@@ -107,17 +106,20 @@ GEM
     concurrent-ruby (1.1.7)
     crass (1.0.6)
     database_cleaner (1.8.5)
-    diff-lcs (1.3)
-    docile (1.3.2)
-    erubi (1.9.0)
+    diff-lcs (1.4.4)
+    docile (1.3.5)
+    erubi (1.10.0)
     execjs (2.7.0)
-    faraday (1.0.1)
+    faraday (1.3.0)
+      faraday-net_http (~> 1.0)
       multipart-post (>= 1.2, < 3)
-    ffi (1.13.1)
+      ruby2_keywords
+    faraday-net_http (1.0.1)
```

Comment:
should faraday-net_http be in twice?

Reply:
I think it's at two different levels; once in its own right, and once as a dependency of something else (`faraday`, in fact)